app/src/main/java/org/simple/clinic/overdue/AppointmentRepository.kt

```diff
@@ -19,7 +20,8 @@ class AppointmentRepository @Inject constructor(
     private val appointmentDao: Appointment.RoomDao,
     private val overdueDao: OverdueAppointment.RoomDao,
     private val userSession: UserSession,
-    private val facilityRepository: FacilityRepository
+    private val facilityRepository: FacilityRepository,
+    private val clock: Clock
```

Comment:
Do you plan to make use of this clock in tests?

Reply:
Not using as of now. Any reason for asking?

Reply:
Just FYI, we should have switched to using Clock everywhere, whoever introduced it first. We now have a mix of ZoneOffset.UTC and Clock.

Reply:
`Clock` is useful for testing, so it's not used everywhere right now. Making an app-wide change would have taken time, but it was needed for testing in one place.